cpp/tests/test_odeseir.cpp

```diff
@@ -150,6 +150,90 @@ TEST(TestSeir, check_constraints_parameters)
     model.parameters.set<mio::oseir::TimeInfected>(6);
     model.parameters.set<mio::oseir::TransmissionProbabilityOnContact>(10.);
     ASSERT_EQ(model.parameters.check_constraints(), 1);
-    
+
```

Comment:
```suggestion
```

Reply:
I can change that but that is not a method I wrote